app/views/api/v1/plans/_show.json.jbuilder

```diff
@@ -34,32 +34,34 @@ if presenter.data_contact.present?
   end
 end
 
-if presenter.contributors.any?
-  json.contributor presenter.contributors do |contributor|
-    json.partial! "api/v1/contributors/show", contributor: contributor,
-                                              is_contact: false
+unless @minimal
```

Comment:
nice. makes a lot of sense to return just enough info for this one to keep the response size down